test/interpret002/run

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+chmod a-r unreadable.txt
```

Comment:
I think this only might work with Linux. I am unsure how this could be made to work with Windows which is currently failing. @melted Any ideas? :)